### PR TITLE
Fix disable child plugins

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: djangocms_text_ckeditor/models.py [hunk 1]
```diff
@@ -25,6 +25,9 @@ class AbstractText(CMSPlugin):
     body = models.TextField(_("body"))
 
     search_fields = ('body',)
+
+    # This property is deprecated. And will be removed in a future release.
+    # It should be set on the Plugin, not the model.
     disable_child_plugins = True
```
Comment (yakky) [inline djangocms_text_ckeditor/models.py]: I don't think we need a deprecation period for this. This is virtually not working now, right?
Anyway the API is on the CMS side, text plugin is just a consumer, thus we can drop this without further warnings

Comment (mkoistinen) [inline djangocms_text_ckeditor/models.py]: Without the deprecation period, this plugin will not work with old CMS versions.

Comment (yakky) [inline djangocms_text_ckeditor/models.py]: You're right :)

**Comment by yakky** [comment]
LGTM

**Pull request merged**